overview/content-types/demo-instructions.nix

```diff
@@ -87,7 +97,7 @@ in
           ];
         }
         {
-          platform = "Ubuntu 24.04";
+          platform = "Ubuntu 24.04/24.10";
```

Comment:
`Ubuntu 24.10` is EOL and as thus we can't test it, that's why [we dropped it](https://github.com/eljamm/ngipkgs/commit/8f5b5d52721407278e4e24e3382f6e35e6c4bfa2).